src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -0,0 +1,266 @@
+                type: CONST.PAYMENT_METHODS.BANK_ACCOUNT,
+            },
+        ];
+
+        this.transferBalance = this.transferBalance.bind(this);
+
+        const selectedAccount = this.getSelectedAccount();
+        PaymentMethods.startWalletTransfer(
+            this.props.userWallet.currentBalance - Fee,
+            selectedAccount ? selectedAccount.id : '',
+        );
+    }
+
+    // eslint-disable-next-line valid-jsdoc
+    /**
+     * Get the selected/default Account for wallet tsransfer
+     * @typedef {import('../../../libs/PaymentUtils').PaymentMethod} PaymentMethod
+     * @returns {PaymentMethod|undefined}
+     */
+    getSelectedAccount() {
+        const paymentMethods = PaymentUtils.getPaymentMethodsList(
+            this.props.bankAccountList,
+            this.props.cardList,
+        );
+
+        const defaultAccount = _.find(
+            paymentMethods,
+            method => method.id === lodashGet(this.props, 'userWallet.walletLinkedAccountID', ''),
+        );
+        const selectedAccount = this.props.walletTransfer.selectedAccountID
+            ? _.find(
+                paymentMethods,
+                method => method.id === this.props.walletTransfer.selectedAccountID,
+            )
+            : defaultAccount;
+
+        return selectedAccount;
+    }
+
+    /**
+     * Transfer Wallet balance
+     * @param {PaymentMethod} selectedAccount
+     */
+    transferBalance(selectedAccount) {
+        if (!selectedAccount) {
+            return;
+        }
+        PaymentMethods.transferWalletBalance(selectedAccount);
+    }
+
+    render() {
+        const selectedAccount = this.getSelectedAccount();
+
+        const selectedPaymentType = selectedAccount && selectedAccount.type === CONST.PAYMENT_METHODS.BANK_ACCOUNT
+            ? CONST.WALLET.TRANSFER_METHOD_TYPE.ACH
+            : CONST.WALLET.TRANSFER_METHOD_TYPE.INSTANT;
+        const transferAmount = (this.props.userWallet.currentBalance - Fee).toFixed(2);
+        const canTransfer = transferAmount > Fee;
+        const isButtonDisabled = !canTransfer || !selectedAccount;
+
+        if (selectedAccount) {
+            const iconProperties = PaymentUtils.getPaymentMethodIconProperties(selectedAccount);
+            selectedAccount.icon = iconProperties.icon;
+            selectedAccount.iconSize = iconProperties.iconSize;
+        }
+
+        return (
+            <ScreenWrapper>
+                <KeyboardAvoidingView>
+                    <HeaderWithCloseButton
+                        title={this.props.translate('common.transferBalance')}
+                        shouldShowBackButton
+                        onBackButtonPress={() => Navigation.goBack()}
+                        onCloseButtonPress={() => Navigation.dismissModal(true)}
+                    />
+                    <View style={[styles.flex1, styles.flexBasisAuto, styles.justifyContentCenter]}>
+                        <CurrentWalletBalance balanceStyles={[styles.text7XLarge]} />
```

Comment:
Where are we getting `7X` from? First time I've seen a number next to a font style.